concordium-sdk/src/main/java/com/concordium/sdk/responses/peerList/Peer.java

```diff
@@ -0,0 +1,30 @@
+package com.concordium.sdk.responses.peerList;
+
+import com.concordium.sdk.types.UInt16;
+import concordium.ConcordiumP2PRpc;
+import inet.ipaddr.IPAddress;
+import inet.ipaddr.IPAddressString;
+import lombok.Builder;
+import lombok.Getter;
+
+/**
+ * Represents the parsed form of {@link ConcordiumP2PRpc.PeerElement}
+ */
+@Builder
+@Getter
+public class Peer {
+
+    private final String nodeId;
+    private final UInt16 port;
```

Comment:
Please add `toString` on the `Uint16.java`.

Reply:
done